.travis.yml

```diff
@@ -25,6 +25,3 @@ jobs:
         - DB=MYSQL
         - REQUIRE_INSTALLER="4.x-dev"
         - PHPUNIT_TEST=1
-        - COMPOSER_INSTALL_ARG=--ignore-platform-reqs
```

Comment:
Did you mean to remove this line as well? If so, can you explain what the intention is there?

Reply:
Yup, originally we needed that line when things weren't installable due to the not being php compatible.  Things are compatible now so we no longer need it